scripts/measures/lib/validate-change-requests.ts

```diff
@@ -47,17 +48,18 @@ interface Cost_MeasuresChange extends baseMeasuresChange {
     submissionMethods?: string[],
 };
 
-interface Quality_MeasuresChange extends baseMeasuresChange {
+interface Base_Quality_MeasuresChange extends baseMeasuresChange {
     nqfId?: string,
+    nationalQualityStrategyDomain?: string,
     isHighPriority?: boolean,
     isInverse?: boolean,
     isRiskAdjusted?: boolean,
     primarySteward?: string,
-    allowedVendors?: string[],
+    measureType?: string,
     allowedPrograms?: string[],
     eMeasureId?: string,
     nqfEMeasureId?: string,
-    measureSets?: string[],
+    isRegistryMeasure?: boolean,
```

Comment:
`allowedVendors` and `measureSets` removed from the Base to be validated more specifically.
`measureType` and `isRegistryMeasure` add since they were missed earlier.